src/utils.js

```diff
@@ -57,6 +57,7 @@ export function getDefaultRegistry() {
   return {
     fields: require("./components/fields").default,
     widgets: require("./components/widgets").default,
+    FieldTemplate: require("./components/templates/DefaultTemplate").default,
```

Comment:
https://github.com/mozilla-services/react-jsonschema-form/blob/e0192b75d24024370d52a43293931c8ddd769752/src/components/Form.js#L162

I think the form doesn't actually receive the template.

Reply:
You're right -- @davidbarratt can you fix this?